pkg/mutation/assignmeta_mutator.go

```diff
@@ -82,6 +82,18 @@ func (m *AssignMetadataMutator) DeepCopy() Mutator {
 	return res
 }
 
+func (m *AssignMetadataMutator) Path() *parser.Path {
+	return m.path
```

Comment:
You need to copy the path in the deepcopy method

Reply:
Done